lib/presto-client/index.js

```diff
@@ -198,6 +198,10 @@ Client.prototype.statementResource = function(opts) {
     if (opts.schema || this.schema) {
         header[client.headers.SCHEMA] = opts.schema || this.schema;
     }
+    if (opts.prepares) {
+        header[client.headers.PREPARE] = opts.prepares.map((s, index) => 'query' + index + '=' + s).join(',');
```

Comment:
Could you add a check if `client.headers.PREPARE` is not defined (for Presto not Trino)?